NextionFontEditor/ZiLib/FileVersion/V5/ZiFontV5.cs

```diff
@@ -38,6 +38,8 @@ public class ZiFontV5 : IZiFont {
         public uint VariableDataLength { get; set; }
         public uint CharDataLength { get; set; }
 
+        public Character[] Characters { get; set; } = new Character[0];
```

Comment:
You can simplify things a lot by using a `List<Character>` instead of the `Character[]`-array. The generic `List<>`-type has methods for adding, removing and inserting items.

Reply:
This was part of an exercise in performance tuning. I read somewhere that Array was a factor faster than List... but also that optimization too early is not a good idea either. So I'm not sure Arrays benefit that much anyway.

I'll change this back to List. The Character class is now upgraded to a proper IZiCharacter interface which has built in performance enhancements.